.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -0,0 +1,18 @@
+---
+name: EPIC
+about: Create a new EPIC issue
+title: "[EPIC]: "
```

Comment:
```suggestion
title: "[EPIC] Your Title"
```
If we want to keep the label in the title, removing the `:` makes it lighter I think.